plugins/flatpak/gs-flatpak.c

```diff
@@ -1301,6 +1320,85 @@ gs_flatpak_app_install_source (GsFlatpak *self, GsApp *app,
 	return TRUE;
 }
 
+static GsApp *
+get_main_app_of_related (GsFlatpak *self,
+			 GsApp *related_app,
+			 GCancellable *cancellable,
+			 GError **error)
+{
+	g_autofree gchar *main_app_name = g_strdup (gs_flatpak_app_get_ref_name (related_app));
```

Comment:
This variable is unused

Reply:
Sorry. That was for a previous version where I had to put a null char in the string directly, but I think the compiler for some reason doesn't warn me about unused variables when they have `g_auto...`.